modules/updater.py

```
@@ -0,0 +1,377 @@
+                webbrowser.open(self._release_url)
+                self.accept()
+                return
+            reply = QMessageBox.question(
+                self, "Ready to Install",
+                f"{self._latest_version} downloaded.\n\n"
+                "Honey Batchr will close and the installer will launch.\nReady to install?",
+                QMessageBox.StandardButton.Yes | QMessageBox.StandardButton.No,
+            )
+            if reply == QMessageBox.StandardButton.Yes:
+                try:
+                    subprocess.Popen([path])
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)